src/commcare_cloud/commands/deploy/formplayer.py

```diff
@@ -0,0 +1,256 @@
+from collections import namedtuple
+from datetime import datetime
+
+import pytz
+import requests
+from dateutil import parser
+from requests import RequestException
+
+from commcare_cloud.alias import commcare_cloud
+from commcare_cloud.cli_utils import ask
+from commcare_cloud.colors import color_warning, color_notice, color_summary
+from commcare_cloud.commands.ansible import ansible_playbook
+from commcare_cloud.commands.ansible.helpers import AnsibleContext
+from commcare_cloud.commands.deploy.sentry import update_sentry_post_deploy
+from commcare_cloud.commands.terraform.aws import get_default_username
+from commcare_cloud.commands.utils import timeago
+from commcare_cloud.fab.deploy_diff import DeployDiff
+from commcare_cloud.fab.git_repo import get_github, github_auth_provided
```

Comment:
I agree it's not ideal but I'm leaving it out of scope for this PR since it requires more significant changes to the API for getting a GitHub repo. Also considering @esoergel comment [here](https://github.com/dimagi/commcare-cloud/pull/4623#discussion_r609074897) I think we should consider in more broard github auth rework.

Reply:
I could not tell if it was safe to reference `fabric.state.env` outside of a fabric context, but it seems important to determine if it is safe for this PR since you are introducing a new workflow where formplayer deploy (done with ansible) is using things from HQ deploy (done with fabric) that reference the fabric context. It looks like the only attribute accessed in `commcare_cloud.fab.git_repo` is `tag_deploy_commits`, but I'm uncertain of how that value is calculated.

Reply:
`fabric.state.env` is just a global dictionary (that supports attribute access). It is safe to use as a normal python object.